webpack/assets/javascripts/react_app/components/common/SkeletonLoader/index.js

```diff
@@ -37,7 +37,7 @@ SkeletonLoader.propTypes = {
 
 SkeletonLoader.defaultProps = {
   skeletonProps: {},
-  emptyState: __('N/A'),
+  emptyState: <DefaultLoaderEmptyState />,
```

Comment:
Would all places that use the skeleton loader have enough room for "Not Available"? I'm for using the same everywhere, but since right now the default is "N/A" this change might cause layout issues elsewhere?

Reply:
Because this might affect other components,  let's keep it `N/A` ,  the DetailsCard can use   ` <DefaultLoaderEmptyState />` instead